paying_for_college/templates/worksheet.html

```diff
@@ -1410,7 +1689,7 @@ <h4 class="aid-form_summary-heading">
                                             $
                                         </span>
                                         <span class="line-item_amount"
-                                        data-financial="totalRepayment"
+                                        data-financial="totalDebt"
```

Comment:
All the variable changes are there to reflect the names of variables in `student-debt-calc`.

I get what you're saying, but we're dealing with a million different things called "total" and "debt". We can change the variable name in `student-debt-calc` and trickle it down, but I think matching variable names to copy only works until the copy changes. :smile: We can discuss it further not-in-the-comments-section. :grin: